### PR TITLE
Add highstock + demo

### PR DESCRIPTION
**Pull request by kirjs**
#11 and #13

### CODE REVIEW

**Comment by kirjs** [comment]
Not sure why the tests are even running as they are in a separate PR

File: src/More.jsx [hunk 1]
```diff
@@ -1,2 +1,2 @@
 module.exports = require('./Highcharts.jsx');
-require("exports?Highcharts!highcharts-more");
\ No newline at end of file
+require("exports?Highcharts!higgihcharts-more");
```
Comment (joshhornby) [inline src/More.jsx]: Guessing this should be `highcharts-more`

Comment (kirjs) [inline src/More.jsx]: Oh, good catch, how did this even get it

Comment (joshhornby) [inline src/More.jsx]: Apart from that looks good to me! 

giphy

**Pull request merged**